### PR TITLE
[krew] Fix krew plugin generator.

### PR DESCRIPTION
**Pull request by mbobrovskyi**
#### What type of PR is this?
/kind bug

#### What this PR does / why we need it:
Fix krew plugin generator.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Review by tenzen-y — COMMENTED** [review]
I forgot to submit my review comments...

File: hack/generate_krew.sh [hunk 1]
```diff
@@ -31,10 +31,7 @@ function generate_platform {
         arch: "${2}"
     uri: https://github.com/kubernetes-sigs/kueue/releases/download/v${VERSION}/kubectl-kueue-${1}-${2}.tar.gz
     sha256: $(curl -L https://github.com/kubernetes-sigs/kueue/releases/download/v${VERSION}/kubectl-kueue-${1}-${2}.tar.gz | shasum -a 256 - | awk '{print $1}')
-    bin: "${3}"
-    files:
-    - from: "*"
-      to: "."
```
[the remark this answers is not in the document]
Comment (mbobrovskyi) [reply hack/generate_krew.sh:37]: This values used by default. So no need to add it.

**Comment by mbobrovskyi** [comment]
/close

Due to optimised and fixed it on #2654.

File: hack/generate_krew.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (tenzen-y) [reply hack/generate_krew.sh:37]: I see. Thanks.

**Pull request closed without merging**